documentation/smg/08_setting_up_jenkins_ci.md

```diff
@@ -0,0 +1,126 @@
+# Setting Up Jenkins CI
+
+We use machines administered by [ANVIL](https://anvil.softeng-support.ac.uk/)
```

Comment:
As a style thing I'd drop "we" from these (e.g. "Our continuous integration (CI) jobs are run on machines administered by ....")